src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -1206,13 +1196,7 @@ def _update_dict(dict1, dict2):
 
 
 def aks_browse(cmd, client, resource_group_name, name, disable_browser=False):
-    """
-    Open a web browser to the dashboard for a managed Kubernetes cluster.
-
-    :param disable_browser: If true, don't launch a web browser after establishing the
-     kubectl port-forward
-    :type disable_browser: bool
-    """
+    """Open a web browser to the dashboard for a managed Kubernetes cluster."""
```

Comment:
You can remove this since it's already defined in help.py.

Reply:
I know this string is redundant (the one in _help.py wins), but it felt wrong to leave the code with no docstring at all. I can remove them if you'd rather have that.

Reply:
Okay that's fine. Up to you.

Reply:
We actually do prefer you remove them.  The reason is we don't want to give anyone the impression that we encourage pulling help from custom command docstrings.

Reply:
Makes sense to me. I'll remove them.